pom.xml

```diff
@@ -5346,7 +5346,13 @@
                 <recipe>tech.picnic.errorprone.refasterrules.BigDecimalRulesRecipes</recipe>
                 <recipe>tech.picnic.errorprone.refasterrules.CharSequenceRulesRecipes</recipe>
                 <recipe>tech.picnic.errorprone.refasterrules.ClassRulesRecipes</recipe>
+                <recipe>tech.picnic.errorprone.refasterrules.CollectionRulesRecipes</recipe>
+                <recipe>tech.picnic.errorprone.refasterrules.ComparatorRulesRecipes</recipe>
```

Comment:
sould we extract other unrealted reciped?

Assuming we need to extend build timeout soon.

Reply:
We we reach limit, we should split recipes by provider and use different profiles for each recipe group. CI execution should stay under 10 minutes.

Reply:
Yes will extract in dedicated PR.